pkg/controller/utils/clientwrapper.go

```diff
@@ -28,8 +28,9 @@ var (
 		[]string{"controller", "method", "resource", "remote", "status"},
 	)
 	metricKubeClientRequestSeconds = prometheus.NewHistogramVec(prometheus.HistogramOpts{
-		Name: "hive_kube_client_request_seconds",
-		Help: "Length of time for kubernetes client requests.",
+		Name:    "hive_kube_client_request_seconds",
+		Help:    "Length of time for kubernetes client requests.",
+		Buckets: []float64{0.05, 0.1, 0.5, 1, 5, 10, 30, 60, 120},
```

Comment:
what is the effect of changing these buckets in existing data that is available in the monitoring system? if there are any existing dashboards that use previous histogram buckets they are probably going to be wrong ot invalid?

also any reason why we chose these specific values?

Reply:
Suspect it's dropped completely similar to what happens if you alter labels. This should be fine as we have very little historical data regardless. (a week or two)

Reply:
As for the values they're just a rough feel for what I'm interested in seeing and our upper range.